statestorage/statestorage_mgr.go

```diff
@@ -55,25 +55,28 @@ func fileExists(filename string) bool {
 	return !info.IsDir()
 }
 
-func InitDb(dbFile string, isNewDB bool) {
-	if (isNewDB == true && fileExists(dbFile)) {
```

Comment:
(old code)
If newDB is true then the file should not (does not) exist -- so we do not take this branch.